src/main/webapp/app/shared/achievement/item/achievement-item.component.html

```diff
@@ -11,7 +11,7 @@
 <div class="d-flex flex-column achievement-item" (click)="selectItem($event)"
      popoverClass="score-popover"
      [ngClass]="[active ? 'active' : '', type, itemStatusCssClass]"
-     [ngbPopover]="popoverContent" #popover="ngbPopover" triggers="manual"
+     [ngbPopover]="popoverContent" #popover="ngbPopover" [autoClose]="false" triggers="manual"
      (mouseenter)="onPopupEnter()" (mouseleave)="onPopupLeave()" placement="top" container="body">
     <div class="badge-content">
         <span *ngIf="item.instantMultiplier >= 0 && active" class="badge badge-pill multiplier-badge" (click)="toggleEditMode()">
```

Comment:
Fixes JS errors prohibiting your fix from working. An adjustment in line 85 is also needed: https://github.com/iteratec/iteratec-teamdojo/pull/74/files#diff-164e0613e6c679bcbd2587cb303b99a3R81
```suggestion
        <span *ngIf="item.instantMultiplier >= 0 && active" class="badge badge-pill multiplier-badge" (click)="toggleEditMode($event)">
```